frontend/src/types.ts

```diff
@@ -68,10 +68,12 @@ export type NotebookResources = {
   requests?: {
     cpu?: string;
     memory?: string;
+    'nvidia.com/gpu'?: number;
```

Comment:
These are strings in my case 🤔 I wonder if we should make sure to covert to number from the override config setting. It worked on clusters with GPUs I think -- so we should be _fine_ for now.

Could you define a "special type" for GPU?
```
/** 
 * OdhDashboardConfig contains gpuSetting as a string value override -- proper gpus return as numbers
 * TODO: Look to make it just number by properly parsing the value
 */
type GPUCount = string | number;
```
```suggestion
    'nvidia.com/gpu'?: GPUCount;
```

That way we can document it once and look to change it in the future.